applications/FSIApplication/python_scripts/partitioned_fsi_base_solver.py

```diff
@@ -282,8 +292,8 @@ def SolveSolutionStep(self):
             KratosMultiphysics.MESH_VELOCITY,
             KratosMultiphysics.FSI_INTERFACE_MESH_RESIDUAL,
             "nodal")
-        self._PrintInfoOnRankZero("","\tNL residual norm: ", nl_res_norm)
-        self._PrintInfoOnRankZero("","\tMesh residual norm: ", mesh_res_norm)
+        KratosMultiphysics.Logger.PrintInfo("","\tNL residual norm: ", nl_res_norm)
+        KratosMultiphysics.Logger.PrintInfo("","\tMesh residual norm: ", mesh_res_norm)
```

Comment:
```suggestion
        KratosMultiphysics.Logger.PrintInfo("::[PartitionedFSIBaseSolver]::","\tMesh residual norm: ", mesh_res_norm)
```

Seems like you often don't use the label, not sure why

Reply:
In this case I just wanted to keep the same output as we have right now. As this is printed after each NL iteration, I think that printing the label might be a bit annoying.